### ISSUE TITLE
Within Varrock, many female NPC's are missing their jaws

### ISSUE THREAD

**Issue by Diavir** [MatthewMariner/lively-cities#9]
### What happened

<img width="215" height="225" alt="Image" src="https://github.com/user-attachments/assets/00b1e6ee-ed63-4531-a1e2-551d0a7ca0a1" />

<img width="347" height="342" alt="Image" src="https://github.com/user-attachments/assets/b4877482-8731-48a6-8ca4-6e275a454052" />

### What you expected instead

Them to have jaws

### Where were you standing?

Varrock square for the H.A.M. member, near the bench for the East bank for the other

### Crowd density

Full (default)

### Render distance

25

### Is RuneLite's Animation Smoothing plugin enabled?

No

### Client log

```text

```

### Before submitting

- [x] I checked the [Known limitations](../../blob/main/README.md#known-limitations) section — some of this is documented behaviour
